src/poliastro/twobody/orbit.py

```diff
@@ -681,6 +681,99 @@ def geostationary(
         altitude = geo_radius - attractor.R
         return cls.circular(attractor, altitude)
 
+    @classmethod
+    def heliosynchronous(
+        cls,
+        a=None,
+        ecc=None,
+        inc=None,
+        raan=0 * u.deg,
```

Comment:
For orbit designers it's more interesting to set the LTAN/LTDN, Local Time on Ascending/Descending Node. There's some working, but incomplete, logic here:

https://github.com/satellogic/orbit-predictor/blob/e05e847e5344f8099a6ddc444cb2c6dced919975/orbit_predictor/predictors/numerical.py#L180-L183